package.json

```diff
@@ -7,6 +7,9 @@
     "type": "git",
     "url": "git+https://github.com/across-protocol/across-smart-contracts-v2.git"
   },
+  "engines": {
+    "node": ">=8.3.0"
```

Comment:
was this an issue for you?

Reply:
@nicholaspai 
The package builds just fine, but ESLint shows linting error `Rest/spread properties are not supported until Node.js 8.3.0. The configured version range is '>=8.0.0'`